src/components/MultiHopTrade/components/TradeConfirm/TradeConfirm.tsx

```diff
@@ -124,7 +124,7 @@ export const TradeConfirm = () => {
   const swapperName = useAppSelector(selectActiveSwapperName)
   const defaultFeeAsset = useAppSelector(selectFirstHopSellFeeAsset)
   const netBuyAmountCryptoPrecision = useAppSelector(selectNetBuyAmountCryptoPrecision)
-  const slippage = useAppSelector(selectSlippagePercentage)
+  const slippage = useAppSelector(selectQuoteOrDefaultSlippagePercentage)
```

Comment:
Shouldn't this be `selectTradeSlippagePercentage` ?

Reply:
You know what, I believe it should! 🧠

Reply:
4e4734534